### PR TITLE
feat: add styles to landing page

### PR DESCRIPTION
**Pull request by seba-pitra**
add opacity animation between login, register and reset password

### CODE REVIEW

File: client/src/components/Login/Login.jsx [hunk 1]
```diff
@@ -14,11 +14,10 @@ import "./Login.css";
 // sendPasswordResetEmail
```
Comment (seba-pitra) [inline client/src/components/Login/Login.jsx:14]: Captura de pantalla 2023-01-01 020355

File: client/src/components/Recovery/Recovery.jsx [hunk 2]
```diff
@@ -48,49 +48,43 @@ const Recovery = () => {
   };
```
Comment (seba-pitra) [inline client/src/components/Recovery/Recovery.jsx:49]: Captura de pantalla 2023-01-01 020422

File: client/src/components/Registrer/Registrer.css [hunk 3]
```diff
@@ -1,65 +1,90 @@
-.login-errormessage{
-    display: flex;
-    width: 300px;
-    background-color: #202633;
-    color: white
+@import url("https://fonts.googleapis.com/css?family=Numans");
```
Comment (seba-pitra) [inline client/src/components/Registrer/Registrer.css:1]: Captura de pantalla 2023-01-01 020409

**Review by valcoellar — APPROVED** [review]
Fantastic !!

**Review by FranSchlatter — APPROVED** [review]

**Pull request merged**